pydantic/dataclasses.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass as dc
+from .main import validate_model, create_model
+
+
+def pydantic_model(cls):
+    fields = {name: (field.type, field.default)
+              for name, field
+              in cls.__dataclass_fields__.items()}
+    model = create_model('', **fields)
+    return model
+
+
+def dataclass(_cls=None, *, init=True, repr=True, eq=True,
```

Comment:
dataclasses take other arguments eg, `eq`, `order`. I don't think they'll conflict with this, so let's add them here, or at least include `**kwargs`

Reply:
The idea was to use the same signature as the native dataclass, only add validation and maybe some other features of pydantic in the future. This is copied from https://github.com/ericvsmith/dataclasses/blob/d4c87c663feb0617fd8c7e6fe0a58882c4ccd468/dataclasses.py#L935 .

Reply:
sorry, ignore me. I thought you had missed some arguments.